src/interface_r/R/auto_generated_wrappers.R

```diff
@@ -225,191 +225,57 @@ h2o4gpu.gradient_boosting_regressor <- function(
-    copy_X = copy_X,
-    n_jobs = as.integer(n_jobs),
-    n_gpus = as.integer(n_gpus),
-    tol = tol,
-    glm_stop_early = glm_stop_early,
-    glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    verbose = verbose,
-    backend = backend
-  )
-  h2o4gpu_model(model, c("regressor"))
-}
-
-#' @export
-h2o4gpu.logistic_regressor <- function(
-	penalty = "l2",
-	dual = FALSE,
-	tol = 0.01,
-	C = 1.0,
-	fit_intercept = TRUE,
-	intercept_scaling = 1.0,
-	class_weight = NULL,
-	random_state = NULL,
-	solver = "liblinear",
-	max_iter = 5000L,
-	multi_class = "ovr",
-	verbose = 0L,
-	warm_start = FALSE,
-	n_jobs = 1L,
+h2o4gpu.elastic_net <- function(
+	n_threads = NULL,
+	gpu_id = 0L,
 	n_gpus = -1L,
-	glm_stop_early = TRUE,
-	glm_stop_early_error_fraction = 1.0,
-	backend = "auto") {
-
-  model <- h2o4gpu$LogisticRegression(
-    penalty = penalty,
-    dual = dual,
-    tol = tol,
-    C = C,
-    fit_intercept = fit_intercept,
-    intercept_scaling = intercept_scaling,
-    class_weight = class_weight,
-    random_state = as_nullable_integer(random_state),
-    solver = solver,
-    max_iter = as.integer(max_iter),
-    multi_class = multi_class,
-    verbose = as.integer(verbose),
-    warm_start = warm_start,
-    n_jobs = as.integer(n_jobs),
-    n_gpus = as.integer(n_gpus),
-    glm_stop_early = glm_stop_early,
-    glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    backend = backend
-  )
-  h2o4gpu_model(model, c("classifier"))
-}
-
-#' @export
-h2o4gpu.lasso_regressor <- function(
-	alpha = 1.0,
 	fit_intercept = TRUE,
-	normalize = FALSE,
-	precompute = FALSE,
-	copy_X = TRUE,
-	max_iter = 5000L,
+	lambda_min_ratio = 1e-07,
+	n_lambdas = 100L,
+	n_folds = 5L,
+	n_alphas = 5L,
 	tol = 0.01,
-	warm_start = FALSE,
-	positive = FALSE,
-	random_state = NULL,
-	selection = "cyclic",
-	n_gpus = -1L,
+	tol_seek_factor = 0.1,
+	lambda_stop_early = TRUE,
 	glm_stop_early = TRUE,
 	glm_stop_early_error_fraction = 1.0,
-	verbose = FALSE,
-	backend = "auto") {
-
-  model <- h2o4gpu$Lasso(
-    alpha = alpha,
-    fit_intercept = fit_intercept,
-    normalize = normalize,
-    precompute = precompute,
-    copy_X = copy_X,
-    max_iter = as.integer(max_iter),
-    tol = tol,
-    warm_start = warm_start,
-    positive = positive,
-    random_state = as_nullable_integer(random_state),
-    selection = selection,
-    n_gpus = as.integer(n_gpus),
-    glm_stop_early = glm_stop_early,
-    glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    verbose = verbose,
-    backend = backend
-  )
-  h2o4gpu_model(model, c("regressor"))
-}
-
-#' @export
-h2o4gpu.ridge_regressor <- function(
-	alpha = 1.0,
-	fit_intercept = TRUE,
-	normalize = FALSE,
-	copy_X = TRUE,
 	max_iter = 5000L,
-	tol = 0.01,
-	solver = "auto",
-	random_state = NULL,
-	n_gpus = -1L,
-	glm_stop_early = TRUE,
-	glm_stop_early_error_fraction = 1.0,
-	verbose = FALSE,
-	backend = "auto") {
+	verbose = 0L,
+	family = "elasticnet",
```

Comment:
What does `family = "elasticnet"` mean?

Reply:
Regression

Reply:
We talked about this offline, it's the backend ElasticNet function's way of distinguishing between classification and regression.  "elasticnet" means regression and "logistic" means regression (this does not many much sense so we should change this).  More discussion [here](https://github.com/h2oai/h2o4gpu/issues/431).